openpype/hosts/houdini/api/creator_node_shelves.py

```diff
@@ -58,6 +67,33 @@ def create_interactive(creator_identifier):
 
     host = registered_host()
     context = CreateContext(host)
+    creator = context.manual_creators.get(creator_identifier)
+    if not creator:
+        raise RuntimeError("Invalid creator identifier: "
+                           "{}".format(creator_identifier))
+
+    pane = stateutils.activePane(kwargs)
+    if isinstance(pane, hou.NetworkEditor):
+        pwd = pane.pwd()
+        subset_name = creator.get_subset_name(
+            variant=variant,
+            task_name=context.get_current_task_name(),
+            asset_doc=get_asset_by_name(
+                project_name=context.get_current_project_name(),
+                asset_name=context.get_current_asset_name()
+            ),
+            project_name=context.get_current_project_name(),
+            host_name=context.host_name
+        )
+
+        tool_fn = {
+            hou.sopNodeTypeCategory(): soptoolutils.genericTool,
+            hou.cop2NodeTypeCategory(): cop2toolutils.genericTool
+        }.get(pwd.childTypeCategory())
+
+        if tool_fn is not None:
+            out_null = tool_fn(kwargs, "null")
+            out_null.setName("OUT_{}".format(subset_name), unique_name=True)
```

Comment:
I think it's good to discuss whether this logic is in the correct place here. It's relatively hardcoded for all creators which could potentially want to do some other unique behavior for this case.

This logic might make more sense to be done directly in the `Creator.create` and then adding the `kwargs` as `pre_create_data` options so it could do the same filtering here from the houdini tool create scripts. 

However, I implemented it like this for now since it forced me to do less code duplication (or still putting it in the Houdini creator base class, etc.) **But** if the functionality this implements is actually preferred and we want to do more unique things per creator then likely it makes much more sense to implement this on the Creator plug-ins themselves.